.gitignore

```diff
@@ -41,8 +41,8 @@ osmt*.env
 
 # Testing ENV
 # Created at runtime
-/test/postman/osmt.postman_collection.json
```

Comment:
This will now be under source control. Is this expected?

Reply:
Yes. This is generated when ran.

Reply:
All api-related postman collections should be gitignored and excluded from this PR. There are four files in total at present, two generated from the OpenApi specs (v2 & v3) and then two more which are generated once tests are added. Really any `*.postman_collection.json` files in the `test/**` directory and subfolders should be ignored.